dotcom-rendering/src/client/sIndicator.ts

```diff
@@ -0,0 +1,255 @@
+import { maybeSIndicatorCapiKey } from './userFeatures/cookies/sIndicatorCapiKey';
+
+type Content = { id: string; rights: { syndicatable: string } };
+
+/**
+ * This is a port/rewrite of the sIndicator browser extension (https://github.com/guardian/sIndicator)
+ * which aims to display the syndication status of content.
+ *
+ * If the 'GU_SINDICATOR_CAPI_KEY' cookie is defined, via syndication subdomain (see https://github.com/guardian/reuse-content)
+ * then this script is loaded and uses the key defined in that cookie to visually augment pages with syndication status
+ * of the piece and any onward journeys (including fronts etc.).
+ */
+
+const overallRoundelElementId = 'gu-sindicator-roundel';
+
+const displaySIndicatorRoundel = (
+	status: 'hide' | 'loading' | 'syndicatable' | 'not-syndicatable' | 'error',
+): void => {
+	const roundelElement =
+		document.getElementById(overallRoundelElementId) ??
+		(() => {
+			const newElem = document.createElement('div');
+			newElem.id = overallRoundelElementId;
+			document.body.appendChild(newElem);
+			return newElem;
+		})();
+
+	if (status === 'hide') {
+		roundelElement.remove();
+		return;
+	}
+
+	roundelElement.style.position = 'fixed';
+	roundelElement.style.bottom = '20px';
+	roundelElement.style.left = '20px';
+	roundelElement.style.zIndex = '79999';
+	roundelElement.style.overflow = 'hidden';
+	roundelElement.style.padding = '10px';
+	roundelElement.style.fontSize = '18px';
+	roundelElement.style.lineHeight = '23px';
+	roundelElement.style.textAlign = 'center';
+	roundelElement.style.borderRadius = '7px';
+	roundelElement.style.width = '40px';
+	roundelElement.style.height = '40px';
+
+	switch (status) {
+		case 'loading':
+			roundelElement.title = 'Checking syndication status...';
+			roundelElement.style.backgroundColor = 'lightblue';
+			// TODO add a spinner
```

Comment:
are we still planning to do this?

Reply:
good catch, thanks - in the end I just used a ⏳ in the end as no spinner gif was available elsewhere and the SvgSpinner from 'source' is a react component, which we're not using here.